app/assets/images/payment_icons/bread.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-bread">
```

Comment:
Can you put the code everything in one line?

You can follow like the [visa icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

Reply:
Done. Thanks!